monkey/monkey_island/cc/ui/src/components/utils/ServerUtils.tsx

```diff
@@ -19,7 +19,7 @@ export function didAllAgentsShutdown() {
 }
 
 export function getCollectionObject(collectionEndpoint: APIEndpoint, key: string) {
```

Comment:
Can you also confirm that all of these are only called manually? Maybe the boolean parameter should be passed into here instead of set to true for all calls.

Reply:
They are not called manually. I will pass `false` to each of them